CMakeLists.txt

```diff
@@ -47,11 +47,12 @@ option(WITH_TITAN_TOOLS "Build with tools." ON)
 option(TRAVIS "Building in Travis." OFF)
 option(CODE_COVERAGE "Generate code coverage report." OFF)
 
-if (CMAKE_BUILD_TYPE STREQUAL "Release")
+if (CMAKE_COMPILER_IS_GNUCXX)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-array-bounds")
+endif()
+
+if (NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Moving it out is enough. You shouldn't change the old condition.

Reply:
Don't get the point but done

Reply:
Actually, you are right. I got confused before. I'll restore it for you.